src/python/foglamp/translators/omf_translator.py

```diff
@@ -0,0 +1,732 @@
+
+    Retrieves the configuration for :
+        relay_url      - URL           - The URL of the PI Connector to send data to.
+        producer_token - producerToken - The producer token that represents this FogLAMP stream
+        types          - OMFTypes      - A JSON object that contains the OMF type definitions for this stream
+
+    Returns:
+        status: True if successful, False otherwise.
+
+    Raises:
+        Exception: Fails to initialise the plugin
+    """
+
+    global _server_name
+    global _relay_url
+    global _producer_token
+    global _types
+
+    global _sensor_location
+    global _sensor_id
+    global _measurement_id
+
+    global _type_measurement_id
+    global _type_sensor_id
+
+    status = True
+
+    try:
+        # URL
+        _server_name = "WIN-4M7ODKB0RH2"
+        _relay_url   = "http://" + _server_name + ":8118/ingress/messages"
+
+        # producerToken
+        _producer_token = "omf_translator_76"
+
+        # OMFTypes
+        _types = [
+            {
+                "id": _type_sensor_id,
+                "type": "object",
+                "classification": "static",
+                "properties": {
+                    "Name": {
+                        "type": "string",
+                        "isindex": True
+                    },
+                    "Location": {
+                        "type": "string"
+                    }
+                }
+            },
+            {
+                "id": _type_measurement_id,
+                "type": "object",
+                "classification": "dynamic",
+                "properties": {
+                    "Time": {
+                        "format": "date-time",
+                        "type": "string",
+                        "isindex": True
+                    },
+                    "x": {
+                        "type": "number"
+                    },
+                    "y": {
+                        "type": "number"
+                    },
+                    "z": {
+                        "type": "number"
+                    },
+                    "pressure": {
+                        "type": "integer"
+                    },
+                    "lux": {
+                        "type": "integer"
+                    },
+                    "humidity": {
+                        "type": "number"
+                    },
+                    "temperature": {
+                        "type": "number"
+                    },
+                    "object": {
+                        "type": "number"
+                    },
+                    "ambient": {
+                        "type": "number"
+                    }
+
+                }
+            }
+        ]
+
+
+    except Exception as e:
+        status  = False
+        message = _message_list["e000006"].format(e)
+
+        _log.error(message)
+        raise Exception(message)
+
+    return status
+
+
+def debug_msg_write(severityMessage, message):
```

Comment:
severityMessage > severity_message; write a docstring with example usage and output